metron-deployment/packaging/docker/deb-docker/pom.xml

```diff
@@ -90,7 +90,13 @@
                                     </includes>
                                 </resource>
                                 <resource>
-                                    <directory>${metron_dir}/metron-platform/metron-enrichment/target/</directory>
+                                    <directory>${metron_dir}/metron-platform/metron-enrichment/metron-enrichment-common/target/</directory>
```

Comment:
Do we need a deployable artifact for `metron-enrichment-common`?  Wouldn't the deployable bits come from `metron-enrichment-storm` alone?

Reply:
Took me a minute to remember why I did this. It seems like that would be correct, but then I saw that we had metron-parsers-common. I wasn't entirely sure why we need common as a separate deployable unit, but I found that it's what we use for `zk_load_configs.sh`. For enrichment, I put the Zookeeper configs into `metron-enrichment-common` because it could be used independent of `metron-enrichment-storm` (say we were to add Spark support, for instance). So the rpm/deb files will lay down the common bits that will include the enrichment configs. 

Apparently the flux files got mixed up in there, so I'm removing those references.

```
%package        enrichment-common
Summary:        Metron Enrichment Common Files
Group:          Applications/Internet
Provides:       enrichment-common = %{version}

%description    enrichment-common
This package installs the Metron Enrichment Common files

%files          enrichment-common
%defattr(-,root,root,755)
%dir %{metron_root}
%dir %{metron_home}
%dir %{metron_home}/bin
%dir %{metron_home}/config
%dir %{metron_home}/config/zookeeper
%dir %{metron_home}/config/zookeeper/enrichments
%{metron_home}/bin/latency_summarizer.sh
%{metron_home}/config/zookeeper/enrichments/bro.json
%{metron_home}/config/zookeeper/enrichments/snort.json
%{metron_home}/config/zookeeper/enrichments/websphere.json
%{metron_home}/config/zookeeper/enrichments/yaf.json
%{metron_home}/config/zookeeper/enrichments/asa.json
%{metron_home}/flux/enrichment/remote-splitjoin.yaml
%{metron_home}/flux/enrichment/remote-unified.yaml
%attr(0644,root,root) %{metron_home}/lib/metron-enrichment-common-%{full_version}-uber.jar
```
Woops...
```
%{metron_home}/flux/enrichment/remote-splitjoin.yaml
%{metron_home}/flux/enrichment/remote-unified.yaml
```

```
# rpm -qa|grep metron
metron-data-management-0.7.1-201904050407.noarch
metron-parsing-storm-0.7.1-201904050407.noarch
metron-profiler-repl-0.7.1-201904050407.noarch
metron-pcap-0.7.1-201904050407.noarch
metron-config-0.7.1-201904050407.noarch
metron-common-0.7.1-201904050407.noarch
metron-metron-management-0.7.1-201904050407.noarch
metron-parsers-0.7.1-201904050407.noarch
metron-enrichment-0.7.1-201904050407.noarch
metron-profiler-spark-0.7.1-201904050407.noarch
metron-indexing-0.7.1-201904050407.noarch
metron-solr-0.7.1-201904050407.noarch
metron-rest-0.7.1-201904050407.noarch
metron-alerts-0.7.1-201904050407.noarch
metron-performance-0.7.1-201904050407.noarch
metron-parsers-common-0.7.1-201904050407.noarch
metron-profiler-storm-0.7.1-201904050407.noarch
metron-elasticsearch-0.7.1-201904050407.noarch
metron-maas-service-0.7.1-201904050407.noarch
```
```
# grep parsers-common bin/*
bin/zk_load_configs.sh:export JAR=metron-parsers-common-$METRON_VERSION-uber.jar
```